doc/generic/pgf/text-en/pgfmanual-en-tikz-matrices.tex

```diff
@@ -513,11 +513,10 @@ \subsubsection{Cell Styles and Options}
   Feature D &           & $\bullet$  & $\bullet$    & $\bullet$  \\
   Feature E &           &            & $\bullet$    & $\bullet$  \\
   };
-
-\draw[ultra thick, rounded corners=1mm, red] (m-1-3.north west)
-                          rectangle (m-6-3.south east);
-\node[anchor=north,
-   align=center, text=red] at (m-6-3.south) {Popular\\Choice!};
+% Add emphasis on selection by the use of "fit" library
+\node[fit={(m-1-4.north west) (m-6-4.south east)},
+      ultra thick, inner sep=0, rounded corners=1mm,
+      draw=cyan, label={[cyan,align=center]270:Popular\\Choice!}]{};
 \end{tikzpicture}
 \end{codeexample}
```

Comment:
Again I suggest to change some colors. Here a full replacement of the `codeexample`, because I want/need to define a color upfront, so I encapsulated it in braces.

```latex
{
\definecolor{ocher}{RGB}{230,230,150}
\begin{codeexample}[
    preamble={\usetikzlibrary{matrix,fit}},
    pre={\definecolor{ocher}{RGB}{230,230,150}},
]
\begin{tikzpicture}[
  font=\sffamily,
  striped col/.style={column #1/.append style={
        every even row/.style={nodes={fill=ocher}}}},
  head color/.style args={#1/#2}{column #1/.append style={
        row 1/.append style={nodes={fill=#2}}}}
]

\matrix [
   matrix of nodes, nodes in empty cells,
   nodes={text width=2cm, align=center,
          minimum height=1.5em, anchor=center},
   striped col/.list={1,...,5}, % add striped col style to all cols
   column 1/.style={ % Override stripes and modify the feature column
     row 1 column 1/.style={nodes={fill=none, draw=none}},
     nodes={fill=ocher!80!black, inner ysep=0},
   },
   % modify headers first via common styles and then specific colors
   row 1/.style={nodes={text depth=0.2ex, text width=2cm, text=white}},
   head color/.list={2/orange,3/teal,4/cyan,5/magenta}
  ] (m)
  {
            & Basic     & Standard   & Professional & Enterprise \\
  Feature A & $\bullet$ & $\bullet$  & $\bullet$    & $\bullet$  \\
  Feature B & $\bullet$ & $\bullet$  & $\bullet$    & $\bullet$  \\
  Feature C &           &            &              & $\bullet$  \\
  Feature D &           & $\bullet$  & $\bullet$    & $\bullet$  \\
  Feature E &           &            & $\bullet$    & $\bullet$  \\
  };
% Add emphasis on selection by the use of "fit" library
\node[fit={(m-1-4.north west) (m-6-4.south east)},
      ultra thick, inner sep=0, rounded corners=1mm,
      draw=cyan, label={[cyan,align=center]270:Popular\\Choice!}]{};
\end{tikzpicture}
\end{codeexample}
}
```

Reply:
Ah for that you can also multiply the color values with 0.8 to mix with black. I'll add in a bit.

Reply:
I still prefer my color choices. Here an image showing your updated version (left) against mine  (right) as they are shown in the manual.

![image showing ilayn's updated version against Mo-Gul's](https://user-images.githubusercontent.com/17992976/84587074-6250c800-ae1c-11ea-91d1-9e58cdc8d127.PNG)

- As you can/might see, your colors of column 1 and "Standard" are very similar.
- Also the bright lime color doesn't fit in very well (especially at the default `codeexample` background color) an really "burns" in ones eyes.

Reply:
No problem. Could you please review line by line in the github review the changes? I thought I changed to what you suggested already. But apparently missed other details